dev-python/geopandas/geopandas-1.0.1.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+DISTUTILS_USE_PEP517=setuptools
+inherit distutils-r1 optfeature
+
+DESCRIPTION="Python tools for geographic data"
+HOMEPAGE="
+	https://github.com/geopandas/geopandas
+	https://pypi.org/project/geopandas/
+"
+SRC_URI="https://github.com/geopandas/${PN}/archive/refs/tags/v${PV}.tar.gz -> ${P}.gh.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+# upstream order
+RDEPEND="
+	dev-python/numpy[${PYTHON_USEDEP}]
+	>=dev-python/pandas-1.4[${PYTHON_USEDEP}]
+	>=dev-python/shapely-2.0.0[${PYTHON_USEDEP}]
+	>=dev-python/pyproj-3.3.0[${PYTHON_USEDEP}]
+	dev-python/fiona[${PYTHON_USEDEP}]
+"
+distutils_enable_tests pytest
+
+python_test() {
+	local deselect=(
```

Comment:
Here too you can use `EPYTEST_DESELECT` instead.

Reply:
`EPYTEST_IGNORE` takes files, not individual tests like `EPYTEST_DESELECT`.

Reply:
And tests with a `[..]` need to be quoted, otherwise the deselecting is ineffective.